crates/bevy_ui/src/focus.rs

```diff
@@ -35,8 +35,8 @@ use smallvec::SmallVec;
 #[derive(Component, Copy, Clone, Eq, PartialEq, Debug, Reflect, Serialize, Deserialize)]
 #[reflect(Component, Serialize, Deserialize, PartialEq)]
 pub enum Interaction {
-    /// The node has been clicked
-    Clicked,
+    /// The node has been pressed (Note: This does not capture click/press-release action)
```

Comment:
Super minor nit: I'd probably put this on a separate line.

```suggestion
    /// The node has been pressed.
    ///
    /// Note: This does not capture click/press-release action.
```

Reply:
Ez, should be good now